src/authorization/WebAclAuthorizer.ts

```diff
@@ -132,7 +138,7 @@ export class WebAclAuthorizer extends Authorizer {
   /**
    * Returns the acl triples that are relevant for the given identifier.
    * These can either be from a corresponding acl file or an acl file higher up with defaults.
-   * Rethrows any non-NotFoundHttpErrors thrown by the AclManager or ResourceStore.
+   * Rethrows any non-NotFoundHttpErrors thrown by the aclManager or ResourceStore.
```

Comment:
?

Reply:
I probably did a replace-all for something and this was caught in the crossfire 😅

Reply:
Ah no, I just remembered. I actually did this on purpose since there no longer is an AclManager class but only an aclManager variable.